js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
+let newNum = document.getElementById('new-num');
+let numsContainer = document.getElementById('added-nums-container');
+let sumElem = document.getElementById('sum');
+let isOdd = false;
+
+addBtn.addEventListener('click', () => {
+    if (newNum.value === '') {
+        alert('Enter a number, please!');
+        return;
+    }
+
+    isOdd = !isOdd;
+    let currentElem = document.createElement('div');   
```

Comment:
This approach works but it's not very optimal and readable. It can be done with a string template literal. 
Also creating elements directly on existing dom elements and changing attributes may trigger browser rerendering and cause performance issues. This can be worked around with a dom fragment.